lambdas/hyf-sns-slackWebhook/src/handlers/sns-payload-logger.js

```diff
@@ -1,8 +1,83 @@
 /**
  * A Lambda function that logs the payload received from SNS.
  */
+const axios = require("axios").default;
+const aws = require("aws-sdk");
+const db = new aws.DynamoDB();
+
+const tableName = process.env.PERSONS_TABLE;
+
 exports.snsPayloadLoggerHandler = async (event, context) => {
-    // All log statements are written to CloudWatch by default. For more information, see
-    // https://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-logging.html
-    console.info(event);
-}
+  var params = {
+    TableName: tableName,
+    Key: { id: { S: event.Records[0].Sns.Message } }, 
+  };
+
+  const result = await db.getItem(params).promise();
+
+  const roundsText = result["Item"].interviewRounds.S > 1 ? "rounds" : "round";
+
+  const payload = {
+    username: "HYF Alumni Hired",
+    icon_emoji: ":hyf-new:",
+    blocks: [
+      {
+        type: "section",
+        text: {
+          type: "mrkdwn",
+          text: `*${result["Item"].fullname.S}* from class *${result["Item"].classNr.S}* just landed a job after *${result["Item"].interviewRounds.S}* interview ${roundsText}! :clap:`,
```

Comment:
I think you can make it a bit more clean by creating the text snippet beforehand.